src/NServiceBus.Testing.Fakes/DefaultTestingLoggerFactory.cs

```diff
@@ -0,0 +1,62 @@
+namespace NServiceBus.Testing
+{
+    using System;
+    using System.IO;
+    using Logging;
+
+    class DefaultTestingLoggerFactory : ILoggerFactory
+    {
+        public DefaultTestingLoggerFactory(LogLevel filterLevel, TextWriter textWriter)
+        {
+            this.filterLevel = filterLevel;
+            textWriterLogger = new TextWriterLogger(textWriter);
+            isDebugEnabled = LogLevel.Debug >= filterLevel;
+            isInfoEnabled = LogLevel.Info >= filterLevel;
+            isWarnEnabled = LogLevel.Warn >= filterLevel;
+            isErrorEnabled = LogLevel.Error >= filterLevel;
+            isFatalEnabled = LogLevel.Fatal >= filterLevel;
+        }
+
+        public ILog GetLogger(Type type)
+        {
+            return GetLogger(type.FullName);
+        }
+
+        public ILog GetLogger(string name)
+        {
+            return new NamedLogger(name, this)
+            {
+                IsDebugEnabled = isDebugEnabled,
+                IsInfoEnabled = isInfoEnabled,
+                IsWarnEnabled = isWarnEnabled,
+                IsErrorEnabled = isErrorEnabled,
+                IsFatalEnabled = isFatalEnabled
+            };
+        }
+
+        public void Write(string name, LogLevel messageLevel, string message)
+        {
+            if (messageLevel < filterLevel)
+            {
+                return;
+            }
+            var datePart = DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss.fff");
```

Comment:
This is a copy paste from the default impl in the core. My idea was to align it with that since this is what happens if you don't specify a specific logger. So I'd say why not?